lib/suspenders/generators/app_generator.rb

```diff
@@ -71,9 +72,7 @@ def customize_gemfile
     def setup_database
       say 'Setting up database'
 
-      if 'postgresql' == options[:database]
-        build :use_postgres_config_template
-      end
+      build :use_postgres_config_template if 'postgresql' == options[:database]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.